File: mysql/tutorials/fastapi/src/connection.py
from typing import List
from pymysql import connect

class Connection:
  def __init__(self) -> None:
    self.__conn = connect(
      host='127.0.0.1',
      port=5001,
      user='root',
      passwd='password',
      database='service'
    )

  def show_tables(self) -> List[str]:
    query = 'SHOW TABLES;'
    curs = self.__conn.cursor()
    curs.execute(query)
    data = curs.fetchall()
    curs.close()
    return [table[0] for table in data]

File: mysql/tutorials/express/src/connection.ts
import { Application } from 'express';
import mysql from 'mysql2';

// Create MySQL Connection
const connection = mysql.createConnection({
  host: '127.0.0.1',
  port: 5001,
  user: 'root',
  password: 'password',
  database: 'service',
});

// MySQL Connect Function
export const connect = (app: Application, port: number) => {
  const query = `SHOW TABLES;`;
  connection
    .promise()
    .query(query)
    .then(() => {
      const log = () => console.log(`Server Running on port ${port}`);
      app.listen(port, log);
    })
    .catch(() => console.log('MySQL Connection Error!'));
};
